assists/i.sh

```diff
@@ -0,0 +1,19 @@
+#/usr/bin/dash
+
+# assist: search man pages
+page="$(
+	ls $(
+		man -w |
+		sed 's/:/\/man\* /g'
+	)/man* |
+	sed '/\//d; s/\.gz//g' |
+	awk '!a[$0]++' |\
+	instantmenu -p "       " -q 'search man pages' -l 12 -h -1
+)"
+
+[ -z "$page" ] && exit 0
+
+for i in $page
```

Comment:
This for-loop will potentially open a lot of terminals, bombing the users system. Enforce a sensible limit here.

Reply:
The for loop should only open terminals if select multiple options with control enter, right? Otherwise instantmenu will always only return one line.

Reply:
You a right, appologies.